libmscore/textbase.cpp

```diff
@@ -2574,10 +2578,15 @@ void TextBase::draw(QPainter* p) const
             if (circle())
                   p->drawEllipse(frame);
             else {
-                  int r2 = frameRound();
+                  qreal baseSpatium = MScore::baseStyle().value(Sid::spatium).toDouble();
+                  qreal frameRoundFactor = (spatium()/baseSpatium)/2;
```

Comment:
See above. It is better to have one place which assigns the value to the variable and later modify it by multiplication if necessary.

Reply:
Fixed.